#request
# Set either node_id or node_label
int32 node_id
string node_label
---
#response 
int32[] path_ids
geometry_msgs/Pose[] path_poses
float32 planning_time